src/diffusion_for_multi_scale_molecular_dynamics/models/axl_diffusion_lightning_model.py

```diff
@@ -222,6 +230,11 @@ def _generic_step(
         with corresponding {sigma(t)}, {xt}, {beta(t)} and {a(t)}. Note the :math:`beta(t)` is used to compute the true
         posterior :math:`q(a_{t-1} | a_t, a_0)` and :math:`p_\theta(a_{t-1} | a_t)` in the atom type loss.
 
+        For the lattice parameters, the loss is defined similarly to the coordinate diffusion. The score to approximate
+        is slightly difference due to the change in the variance schedule (variance preserving instead of variance
```

Comment:
`is slightly difference` -> `is slightly different`

Reply:
fixed